src/devices/Media/VideoDevice/samples/VideoDevice.Samples.csproj

```diff
@@ -3,10 +3,12 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net5.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
why this is needed?

Reply:
Ho, it's because this one commit is from another PR, the audio/video one where it's needed. For some reason, it appear here as well.